From a4dc336ad0bc80a9203d7efa154cd919023f30c1 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 9 May 2017 17:39:41 +0800
Subject: [PATCH 1912/5242] MLK-15001-9 media: bus format: Add
 RGB101010_1X7X5_SPWG/JEIDA support

commit  22e2dddd3d6207ef4499841e6e17e0547ff4eb39 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds 30bit RGB101010 LVDS pixel formats support for
the SPWG and JEIDA LVDS mapping standards.  Each pixel is transferred
on 5 lanes with 7bit respectively.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/linux/media-bus-format.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/uapi/linux/media-bus-format.h b/include/uapi/linux/media-bus-format.h
index 166475d..3083f07 100644
--- a/include/uapi/linux/media-bus-format.h
+++ b/include/uapi/linux/media-bus-format.h
@@ -34,7 +34,7 @@
 
 #define MEDIA_BUS_FMT_FIXED			0x0001
 
-/* RGB - next is	0x101d */
+/* RGB - next is	0x101f */
 #define MEDIA_BUS_FMT_RGB444_1X12		0x1016
 #define MEDIA_BUS_FMT_RGB444_2X8_PADHI_BE	0x1001
 #define MEDIA_BUS_FMT_RGB444_2X8_PADHI_LE	0x1002
@@ -63,6 +63,8 @@
 #define MEDIA_BUS_FMT_RGB161616_1X48		0x101a
 #define MEDIA_BUS_FMT_RGB888_1X30_PADLO		0x101b
 #define MEDIA_BUS_FMT_RGB666_1X30_PADLO		0x101c
+#define MEDIA_BUS_FMT_RGB101010_1X7X5_SPWG	0x101d
+#define MEDIA_BUS_FMT_RGB101010_1X7X5_JEIDA	0x101e
 
 /* YUV (including grey) - next is	0x202c */
 #define MEDIA_BUS_FMT_Y8_1X8			0x2001
-- 
1.7.9.5

